neuralmonkey/runners/beamsearch_runner.py

```diff
@@ -34,9 +34,9 @@ def __init__(self,
 
         # We need to define the np.empty arrays here due to the usage
         # of np.append later
-        self._scores = np.empty([0, decoder.beam_size], dtype=float)
-        self._parent_ids = np.empty([0, decoder.beam_size], dtype=int)
-        self._token_ids = np.empty([0, decoder.beam_size], dtype=int)
+        self._scores = np.empty(0)
+        self._parent_ids = np.empty(0)
+        self._token_ids = np.empty(0)
```

Comment:
těchhle pět řádků tu nemusí vůbec bejt, ne?

Reply:
Ted nevim, jestli mypy nebo pylint bude nadavat, ze inicializujes class attributy mimo init.

Reply:
to by neměl, na tom v pythonu neni nic nelegálního

Reply:
No z hlediska citelnosti je to trosicku prasarna. Prece jen chci mrknout do __init__ a hned mit prehled o atributech.

Reply:
tohle jsou privátní proměnný, do těch nikomu nic neni. Tohle neni céčko, abys měl oddělenou deklaraci od definice.

Reply:
čili tohle ještě vyhodit a pak dávám approve